src/examples/s3-storage/src/frontend/types.ts

```diff
@@ -17,8 +17,8 @@ export type Scalars = {
   Boolean: { input: boolean; output: boolean; }
   Int: { input: number; output: number; }
   Float: { input: number; output: number; }
-  /** Media type scalar */
-  Media: { input: any; output: any; }
+  /** Image type scalar */
```

Comment:
These type changes are corrections from the demo. Apologies they got caught up in this PR. This is correct though.